maintenance/manageInactiveWikisV2.php

```diff
@@ -83,6 +78,14 @@ private function checkLastActivity(
 
 		$lastActivityTimestamp = $activity->timestamp;
 
+		$track = ($lastActivityTimestamp != 0) ? 'default': 'no-edits'
```

Comment:
Missing spaces around parenthesis and missing semicolon, and `!==` please?

Reply:
damn php and mediawiki standards..